packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -0,0 +1,254 @@
+import path from 'path'
+import type { Command } from '@prisma/sdk'
+import { arg, format, HelpError, isError, getSchemaPath, link, getCommandWithExecutor } from '@prisma/sdk'
+import chalk from 'chalk'
+import { Migrate } from '../Migrate'
+import { MigrateDiffNeedsPreviewFeatureFlagError } from '../utils/errors'
+import type { EngineArgs, EngineResults } from '../types'
+import Debug from '@prisma/debug'
+
+const debug = Debug('prisma:migrate:diff')
+export class MigrateDiff implements Command {
+  public static new(): MigrateDiff {
+    return new MigrateDiff()
+  }
+
+  private static help = format(`
+${
+  process.platform === 'win32' ? '' : chalk.bold('🔍 ')
+}Compares the database schema from two arbitrary sources, and displays the differences either as a human-readable summary (by default) or an executable script.
```

Comment:
Should we list here already what those supported sources are?

I'd also change "displays" to "outputs"

Reply:
>Should we list here already what those supported sources are?
Too long for a one liner here 😅 Better to keep it short?